scripts/run.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+# SPDX-FileCopyrightText: Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES.
+
+# Install
+set -euo pipefail
+
+# RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CUDA_VERSION})"
+RAPIDS_PY_CUDA_SUFFIX=12
+# TODO: set this to main once dask-cudf is compatible
+# DASK_VERSION=main
+DASK_VERSION=2024.12.1
+export PIP_YES=true
+export PIP_PRE=true
+
+# Should this use nightly wheels or rapids-download-wheels-from-s3?
```

Comment:
This should use nightly wheels like https://anaconda.org/rapidsai-wheels-nightly/cudf-cu12. e.g. `pip install -i https://pypi.anaconda.org/rapidsai-wheels-nightly/simple cudf-cu12`

The `rapids-download-wheels-from-s3` utility is meant to download from downloads.rapids.ai (VPN required). We use S3 artifacts to share wheels/conda packages between build and test jobs. (Build job uploads the artifact, test job downloads the artifact.) Since this workflow is meant to test nightlies, we should test the artifacts published on the `rapidsai-wheels-nightly` package index.